src/EnergyPlus/WeatherManager.cc

```diff
@@ -517,30 +505,30 @@ namespace WeatherManager {
 
     } // clear_state, for unit tests
 
-    void ManageWeather(IOFiles &ioFiles)
+    void ManageWeather(EnergyPlusData& state)
     {
 
         // SUBROUTINE INFORMATION:
         //       AUTHOR         Rick Strand
         //       DATE WRITTEN   May 1997
         //       MODIFIED       June 1997 (general clean-up)
-        //       RE-ENGINEERED  na
 
         // PURPOSE OF THIS SUBROUTINE:
         // This subroutine is the main driver of the weather manager module.
         // It controls the assignment of weather related global variables as
         // well as the reads and writes for weather information.
 
-        // METHODOLOGY EMPLOYED:
-        // Standard EnergyPlus "manager" methodology.
-
-        // FLOW:
-
-        InitializeWeather(ioFiles, PrintEnvrnStamp);
+        InitializeWeather(state.files, PrintEnvrnStamp);
 
+        bool anyEMSRan = false;
+        // Cannot call this during sizing, because EMS will not intialize properly until after simulation kickoff
+        if (!DataGlobals::DoingSizing && !DataGlobals::KickOffSimulation) {
+            EMSManager::ManageEMS(
+                state, DataGlobals::emsCallFromBeginZoneTimestepBeforeSetCurrentWeather, anyEMSRan, ObjexxFCL::Optional_int_const()); // calling point
+        }
```

Comment:
Here's the actual new calling point. Calling this during sizing or before KickOffSimulation results in missing ERL variables, because of input timing issues. So, for now, this calling point won't fire during sizing (noted in the docs). As long as we rely on KickOffSimulation to fire every component for input processing, I don't see a way around this. Unless we move that up before sizing, but that could have bad side effects.